.docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:8.1-cli-bullseye
```

Comment:
It would be nice to be able to define the PHP version through an environment variable.